### PR TITLE
Add a "/blocked" command

### PR DESCRIPTION
**Pull request by SuperDragonXD**
Adds a `/blocked` command telling a user what to do when he/she is blocked.

### CODE REVIEW

**Comment by LunarTwilight** [comment]
Could you do a few fixes first?
- Put a space between the function and it's args (eg `async execute(interaction) {` -> ` async execute (interaction) {`)
- Make spacing use 4 spaces instead of 2
- Remove comma dangle

Thanks!

**Review by LunarTwilight — CHANGES_REQUESTED** [review]
Could you do a few fixes first?
- Put a space between the function and it's args (eg `async execute(interaction) {` -> ` async execute (interaction) {`)
- Make spacing use 4 spaces instead of 2
- Remove comma dangle

Thanks!

**Comment by SuperDragonXD** [comment]
Made those changes

File: commands/blocked.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { MessageActionRow, MessageButton } = require('discord.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('blocked')
+        .setDescription('Gives information on what to do when you are blocked')
```
Comment (LunarTwilight) [inline commands/blocked.js:7]: Missing a comma at the end of this, Maybe also change the text to be "when you (or another user) is blocked"?

File: events/blockedResponse.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                        .setLabel('Contact Fandom staff')
+                        .setStyle('LINK')
+                );
+
+                interaction.reply({
+                    content: `If you are accidentally caught in a global block targeted towards your username or IP address, please contact Fandom staff to remove it.\n\nIn order to effectively locate the block and look into the matter, you must include **all** of the information in the box you see while you're blocked (which can be seen when you try to edit a page), especially the block ID.`,
+                    components: [button],
+                    ephemeral: true
+                });
+
```
Comment (LunarTwilight) [inline events/blockedResponse.js:31]: Include a `break;` here

File: events/blockedResponse.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            case 'blocked-local':
+                let localBlockedMessage = `If you have been blocked on one community, and you do not know why, or you disagree with the reason, __stay calm__. Please do the following steps to appeal your block:
+1. Post a message to your Talk Page or Message Wall, sending a nice message about appealing your block. 
+2. If they disallowed editing your talk or wall, try leaving them a message on [Community Central](https://c.fandom.com).
+3. If they still won't appeal your block, you can simply wait it out (if it's not indefinite), or join another Fandom community.
+
+**Note**: If they are abusing their rights (like blocking anyone who wants to edit), please contact another admin about the issue. **If there are no other admins, please [contact Fandom staff](https://support.fandom.com/hc/en-us/requests/new?ticket_form_id=360000931094&tf_360017834073=__dc.my_account_is_blocked__&tf_1500002786382=fandom_block_local) asap**.`;
+
+                interaction.reply({ content: localBlockedMessage, ephemeral: true });
```
Comment (LunarTwilight) [inline events/blockedResponse.js:40]: split this up onto each line like the one above

**Review by LunarTwilight — CHANGES_REQUESTED** [review]

File: commands/blocked.js (hunk 1 above)
Comment (SuperDragonXD) [reply commands/blocked.js]: Done.

File: events/blockedResponse.js (hunk 3 above)
Comment (SuperDragonXD) [reply events/blockedResponse.js:43]: Done. Should I also separate the `blocked-global` lines too?

Comment (LunarTwilight) [reply events/blockedResponse.js:43]: No, that's fine as is.

**Review by LunarTwilight — APPROVED** [review]
LGTM, I'll run this through the workflow, then run my linter over it, fix any issues, and then deploy. Thanks for the PR.

**Pull request merged**